src/styles/blocks/content.scss

```diff
@@ -0,0 +1,29 @@
+.content {
+
+  &__catalog {
+    display: flex;
+    flex-wrap: wrap;
+
+    width: getCatalogWidth(
+      $card-width,
+      $card-margin-right,
+      $container-padding-horizontal
+    );
+
+    padding: 50px 40px;
```

Comment:
Replace 'padding: 50px 40px;' with 'padding: 50px $container-padding-horizontal;' to use the SCSS variable.